### PR TITLE
Pd 413 review v mware snapshots and storage state behavior docs

### PR DESCRIPTION
**Pull request by DjP-iX**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALEUIReference/DataProtection/VMwareSnapshotsScreen.md [hunk 1]
```diff
@@ -9,20 +9,30 @@ tags:
...
-Click  **VMware Snapshot Integration** to display the **Add VMware Snapshot** screen.
+Use the **VMware Snapshot Integration** option on the **Data Protection > Periodic Snapshot Tasks** widget to create snapshots when you are using TrueNAS SCALE as a VMWare datastore.
+See [Creating VMWare Snapshots]({{< relref "creatingvmwaresnapshots.md" >}}) for a detailed tutorial.
 
-   {{< trueimage src="/images/SCALE/DataProtection/emptyvmwaresnapshotadd.png" alt="Add VMware Snapshot Screen" id="Add VMware Snapshot Screen" >}}
+Click **VMware Snapshot Integration** to open the **VMWare Snapshots** screen.
+
+{{< trueimage src="/images/SCALE/DataProtection/vmwaresnapshottask.png" alt="VMware Snapshot Integration" id="VMware Snapshot Integration" >}}
+
+Click the **Add** button to display the **Add VMware Snapshot** screen.
```
Comment (micjohnson777) [inline content/SCALE/SCALEUIReference/DataProtection/VMwareSnapshotsScreen.md:19]: This is an older method of documenting the ui functions, suggest making these changes to update it to what we are currently doing.
```suggestion
**VMware Snapshot Integration**  opens the **VMWare Snapshots** screen.

{{< trueimage src="/images/SCALE/DataProtection/vmwaresnapshottask.png" alt="VMware Snapshot Integration" id="VMware Snapshot Integration" >}}

**Add** opens the **Add VMware Snapshot** screen.
```

File: content/SCALE/SCALEUIReference/DataProtection/VMwareSnapshotsScreen.md [hunk 2]
```diff
@@ -9,20 +9,30 @@ tags:
...
+| **ZFS Filesystem** | Select a TrueNAS ZFS dataset from the dropdown list of options. This field does not populate until you click **Fetch Datastores**. You must click **Fetch Datastores** before clicking in this field or the creation process fails. |
 {{< /truetable >}}
 
-Click **Fetch DataStores** to connect TrueNAS connects to the VMware host. 
-This synchronizes TrueNAS SCALE with the VMWare host and populates the **ZFS Filesystem** and **Datastore** dropdown lists with the information from the VMware host response.
+Click **Fetch DataStores** to connect TrueNAS SCALE to the VMWare host.
+This synchronizes TrueNAS SCALE with the VMWare host and populates the **ZFS Filesystem** and **Datastore** dropdown lists with the information from TrueNAS and the VMWare host response.
+
+Configured snapshots appear on the **VMware Snapshots** screen.
```
Comment (micjohnson777) [inline content/SCALE/SCALEUIReference/DataProtection/VMwareSnapshotsScreen.md:36]:
```suggestion
Configured snapshots show on the **VMware Snapshots** screen. 
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Caught a few things. Good work on the content changes and improvements!!

File: content/SCALE/SCALETutorials/DataProtection/CreatingVMwareSnapshots.md [hunk 3]
```diff
@@ -9,52 +9,206 @@ tags:
...
 You must have a paid edition of VMWare ESXi to use the TrueNAS SCALE VMWare Snapshots feature.
-If you try to use them with the free-edition of VMware ESXi, you see this error message: "Error, Can't create snapshot, current license or ESXi version prohibits execution of the requested operation."
 ESXi free has a locked (read-only) API that prevents using TrueNAS SCALE VMWare Snapshots.
-The cheapest ESXi edition that is compatible with TrueNAS VMware Snapshots is VMWare vSphere Essentials Kit.
+
+This tutorial uses ESXi version 8.
 {{< /hint >}}
 
+When creating a ZFS snapshot of the connected dataset, VMWare automatically takes a snapshot of any running virtual machine.
+VMware snapshots can integrate VMware Tools, making it possible to quiesce VM snapshots, sync filesystems, take shadow copy snapshots, and more.
```
[the remark this answers is not in the document]
Comment (DjP-iX) [reply content/SCALE/SCALETutorials/DataProtection/CreatingVMwareSnapshots.md:22]: see below

File: content/SCALE/SCALETutorials/DataProtection/CreatingVMwareSnapshots.md [hunk 4]
```diff
@@ -9,52 +9,206 @@ tags:
...
+This allows VMware snapshots to be application-consistent instead of crash-consistent.
```
[the remark this answers is not in the document]
Comment (DjP-iX) [reply content/SCALE/SCALETutorials/DataProtection/CreatingVMwareSnapshots.md:23]: Quiesce and X-consistent are both sort of terms of art when it comes to VMware so they need to be used, but I've added a bit more explanation here.

File: content/SCALE/SCALETutorials/DataProtection/CreatingVMwareSnapshots.md [hunk 5]
```diff
@@ -9,52 +9,206 @@ tags:
...
+      Select **Device** for **Extent Type** and select the zvol from the **Device** dropdown.
+      Leave **Sharing Platform** set to VMware and **Target** set to **Create New**, then click **Next**.
+  
+    b. Set **Portal** to **Create New**.
+        You can leave **Discovery Authentication Method** set to **NONE**, or select **CHAP** or **Mutual CHAP** and enter a **Discovery Authentication Group** ID.
+        Click **Add** next to **IP Address** and select either **0.0.0.0** for IPv4 or **::** for IPv6 to listen on all ports.
+
+    c. Leave **Initiators** blank and click **Save**.
+
+3. In the VMWare ESXi Host Client, go to **Storage**, select **Adapters**, and then click **Software iSCSI** to configure the iSCSI connection.
```
[the remark this answers is not in the document]
Comment (DjP-iX) [reply content/SCALE/SCALETutorials/DataProtection/CreatingVMwareSnapshots.md:60]: ESXi Host Client is the product name

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**